test/wrappers/vroom_test.rb

```diff
@@ -580,4 +578,182 @@ def test_ensure_total_time_and_travel_info_with_vroom
+        },
+        quantities: [{
+          unit_id: 'unit_0',
+          value: 2
+        }]
+      }],
+      configuration: {
+        preprocessing: {
+          prefer_short_segment: true
+        },
+        resolution: {
+          duration: 100
+        },
+        restitution: {
+          intermediate_solutions: false,
+        }
+      }
+    }
+    vrp = TestHelper.create(problem)
+    result = vroom.solve(vrp, 'test')
+    assert result
+    assert_equal(result[:routes][0][:activities].index{ |activity| activity[:pickup_shipment_id] == 'shipment_0' } + 1, result[:routes][0][:activities].index{ |activity| activity[:delivery_shipment_id] == 'shipment_0' })
+    assert_equal(result[:routes][0][:activities].index{ |activity| activity[:pickup_shipment_id] == 'shipment_1' } + 1, result[:routes][0][:activities].index{ |activity| activity[:delivery_shipment_id] == 'shipment_1' })
+    assert_equal 0, result[:unassigned].size
+    assert_equal 6, result[:routes][0][:activities].size
+  end
+
+  def test_mixed_shipments_and_services
+    vroom = OptimizerWrapper.config[:services][:ortools]
+    problem = {
+      matrices: [{
+        id: 'matrix_0',
+        time: [
+          [0, 1, 1, 1],
+          [1, 0, 1, 1],
+          [1, 1, 0, 1],
+          [1, 1, 1, 0]
+        ]
+      }],
+      units: [{
+        id: 'unit_0',
+      }],
+      points: [{
+        id: 'point_0',
+        matrix_index: 0
+      }, {
+        id: 'point_1',
+        matrix_index: 1
+      }, {
+        id: 'point_2',
+        matrix_index: 2
+      }, {
+        id: 'point_3',
+        matrix_index: 3
+      }],
+      vehicles: [{
+        id: 'vehicle_0',
+        cost_time_multiplier: 1,
+        start_point_id: 'point_0',
+        end_point_id: 'point_0',
+        matrix_id: 'matrix_0'
+      }],
+      services: [{
+        id: 'service_1',
+        activity: {
+          point_id: 'point_1',
+        },
+        quantities: [{
+          unit_id: 'unit_0',
+          setup_value: 1,
+        }]
+      }],
+      shipments: [{
+        id: 'shipment_1',
+        pickup: {
+          point_id: 'point_2',
+          duration: 1,
+          late_multiplier: 0,
+        },
+        delivery: {
+          point_id: 'point_3',
+          duration: 1,
+          late_multiplier: 0,
+        }
+      }],
+      configuration: {
+        preprocessing: {
+          prefer_short_segment: true
+        },
+        resolution: {
+          duration: 100
+        },
+        restitution: {
+          intermediate_solutions: false,
+        }
+      }
+    }
+    vrp = TestHelper.create(problem)
+    result = vroom.solve(vrp, 'test')
+    assert result
+    assert result[:routes][0][:activities].index{ |activity| activity[:pickup_shipment_id] == 'shipment_1' } < result[:routes][0][:activities].index{ |activity| activity[:delivery_shipment_id] == 'shipment_1' }
+    assert_equal 0, result[:unassigned].size
+    assert_equal 5, result[:routes][0][:activities].size
+  end
```

Comment:
It looks like we copy these 3 tests from `ortools_test.rb` to here. 

I wonder if we cannot move them to a more centralized location (for example `wrapper_solver_test`?) and call both ortools and vroom one after another inside the test? If not, we can maybe extract the problems to TestHelper (if they are completely the same) so that they will be in one location.

Reply:
Does it seems ok to you to centralize the tests in another PR ? And rework the tests in a more general way ?